### PR TITLE
Fix OBS import

### PR DESCRIPTION
**Pull request by aunger**
Add meta (chapter) contents to OBS imports

---
This change is [Reviewable](https://reviewable.io/reviews/wycliffeassociates/otter-common/125)

### CODE REVIEW

**Review by KJoslyn — APPROVED** [review]
> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/wycliffeassociates/otter-common/125)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @sarabiaj)

File: src/main/kotlin/org/wycliffeassociates/otter/common/domain/resourcecontainer/project/markdown/MarkdownProjectReader.kt [hunk 1]
```diff
@@ -82,7 +83,7 @@ class MarkdownProjectReader(private val isHelp: Boolean) : IProjectReader {
     }
 
     private fun fileToId(f: OtterFile): Int =
-        f.nameWithoutExtension.toIntOrNull() ?: 0
+        f.nameWithoutExtension.toIntOrNull() ?: 1
```
Comment (jsarabia) [inline src/main/kotlin/org/wycliffeassociates/otter/common/domain/resourcecontainer/project/markdown/MarkdownProjectReader.kt:86]: I would expect an id value of 0 means it hasn't been inserted into a database, wouldn't 1 cause problems?

**Review by aunger — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/wycliffeassociates/otter-common/125)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @KJoslyn and @sarabiaj)

___
*[src/main/kotlin/org/wycliffeassociates/otter/common/domain/resourcecontainer/project/markdown/MarkdownProjectReader.kt, line 86 at r1](https://reviewable.io/reviews/wycliffeassociates/otter-common/125#-LnThs3HpWKyM8dPFd8M-r1-86:-LnszxWF27aBcfPXb6pA:bfqv45j) ([raw file](https://github.com/wycliffeassociates/otter-common/blob/daf96677a3f903a9b0947f6f9ef3cc788f22a65c/src/main/kotlin/org/wycliffeassociates/otter/common/domain/resourcecontainer/project/markdown/MarkdownProjectReader.kt#L86)):*
<details><summary><i>Previously, sarabiaj (Joe) wrote…</i></summary><blockquote>

I would expect an id value of 0 means it hasn't been inserted into a database, wouldn't 1 cause problems?
</blockquote></details>

This isn't a db row ID.  I renamed this function to avoid this confusion.

**Review by jsarabia — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/wycliffeassociates/otter-common/125#-:-Lo1rxnl38eQ0S-AFmja:bnfp4nl)* status: 0 of 1 files reviewed, all discussions resolved (waiting on @KJoslyn)

**Pull request merged**